CMakeLists.txt

```
@@ -78,7 +78,7 @@ install(TARGETS maeparser
 
 INSTALL(EXPORT maeparser-targets
     FILE ${PROJECT_NAME}-config.cmake
-    DESTINATION lib/cmake)
+    DESTINATION lib${LIB_SUFFIX}/cmake)
```

I don't know cmake very well... But would it be more idiomatic to use `CMAKE_INSTALL_LIBDIR`, instead?

---

I also do not really know CMAKE well.
The symptom is that unpatched, the file installs to `/usr/lib` on Fedora 64-bit systems.

I am the packer for maeparser on Fedora, and for compliance with our guidelines, the location must be under `/usr/lib64` on 64-bit systems.

The patch, as submitted, accomplishes this goal.

---

I think Dan's suggestion makes sense: right above this block, we set `CMAKE_INSTALL_LIBDIR` as the install directory for libraries in the `maeparser` target. I think it makes sense to put the `cmake` directory there too:
```suggestion
    DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
```

This should also work, and is more coherent with the other definitions.